backend/remote/backend_mock.go

```diff
@@ -960,6 +1050,14 @@ func (m *mockWorkspaces) Read(ctx context.Context, organization, workspace strin
 	return w, nil
 }
 
+func (m *mockWorkspaces) ReadByID(ctx context.Context, workspaceID string) (*tfe.Workspace, error) {
```

Comment:
this stuff looks unrelated

Reply:
Yeah but it's required for valid mocks. I had to update these from upgrading the version of go-tfe.